src/main/java/com/box/sdk/BoxAPIResponse.java

```diff
@@ -27,6 +27,7 @@
     private static final int BUFFER_SIZE = 8192;
 
     private final HttpURLConnection connection;
+    private final Map<String, String> headers;
```

Comment:
Reusing BoxAPIResponse. It's an existing class. No need for special BoxBatchAPIResponse as the response will be same but a list instead of one response